src/cli/cluster/worker.js

```diff
@@ -82,8 +85,15 @@ module.exports = class Worker extends EventEmitter {
   }
 
   onMessage(msg) {
-    if (!_.isArray(msg) || msg[0] !== 'WORKER_BROADCAST') return;
-    this.emit('broadcast', msg[1]);
+    switch (_.isArray(msg) && msg[0]) {
```

Comment:
I really hate this style of mixing conditional logic and expression evaluation on single lines, though I've seen it cropping up more and more in the codebase.  It seems to violate the fail-fast principle and sacrifices expressiveness for succinctness.

That said, if we do a destructuring approach on the args like I recommended above, then this becomes a moot point since the isArray check is no longer necessary.